addons/viewport/src/manager/components/Panel.js

```diff
@@ -170,11 +170,6 @@ export class Panel extends Component {
       this.iframe.style.height = viewport.styles.width;
       this.iframe.style.width = viewport.styles.height;
     }
-
-    // Always make parent's height equals iframe
-    if (this.iframe.parentElement) {
```

Comment:
Why delete this???

Reply:
Because I changed the way the overflow works for the preview